src/main/java/step3/LottoConstant.java

```diff
@@ -0,0 +1,26 @@
+package step3;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoConstant {
+
+    public static String LOTTO_DELIMITER = ",";
+    public static int LOTTO_ELEMENTS_SIZE = 6;
+    public static int LOTTO_PRICE = 1000;
```

Comment:
다음과 같이 자릿수에 `_`를 사용하면 가독성을 높일 수 있어요.
```suggestion
    public static int LOTTO_PRICE = 1_000;
```